Realm/RLMRealm.h

```diff
@@ -233,7 +231,7 @@ typedef void(^RLMNotificationBlock)(NSString *notification, RLMRealm *realm);
  but re-running the query which provided `oldObject` will once again return
  the valid object.
 
- Calling this when not in a write transaction will throw an exception.
+ @warning Calling this when not in a write transaction will throw an exception.
```

Comment:
I'd prefer to stay consistent in our write transaction warnings, and I prefer the "This method can only be called during a write transaction." wording.